sw/device/tests/sival/BUILD

```diff
@@ -8,21 +8,45 @@
 test_suite(
     name = "sv1_tests",
     tests = [
-        "//sw/device/tests:rstmgr_cpu_info_test",
-        "//sw/device/tests:rv_core_ibex_rnd_test",
+        "//sw/device/silicon_creator/rom/e2e/jtag_inject:rom_e2e_openocd_debug_test",
+        "//sw/device/tests:rstmgr_cpu_info_test_fpga_cw310_rom_with_fake_keys",
+        "//sw/device/tests:rstmgr_cpu_info_test_fpga_cw310_sival",
+        "//sw/device/tests:rv_core_ibex_epmp_test_functest_fpga_cw310_sival",
+        "//sw/device/tests:rv_core_ibex_mem_test_functest_fpga_cw310_sival",
+        "//sw/device/tests:rv_core_ibex_rnd_test_fpga_cw310_sival",
     ],
 )
 
 test_suite(
     name = "sv2_tests",
     tests = [
         "//sw/device/tests:aes_new_smoketest",
-        "//sw/device/tests:aon_timer_irq_test",
+        "//sw/device/tests:aon_timer_irq_test_fpga_cw310_sival_rom_ext",
```

Comment:
Can we update the script to skip the `exec_env` suffix? This way we'll be able to run the test suite against FPGA and silicon using different tag filters. Thanks!

Reply:
Done